src/PrestaShopBundle/Controller/Admin/Improve/StoreController.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * 2007-2017 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2017 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShopBundle\Controller\Admin\Improve;
+
+use PrestaShop\PrestaShop\Adapter\Shop\Context;
+use PrestaShopBundle\Controller\Admin\FrameworkBundleAdminController;
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * Responsible of "Improve > Modules > Modules Catalog" page display
+ */
+class StoreController extends FrameworkBundleAdminController
+{
+    /**
+     * @var string The controller name for routing.
+     */
+    const CONTROLLER_NAME = 'AdminAddonsCatalog';
+
+    public function indexAction(Request $request)
+    {
+        return $this->render('@PrestaShop/Admin/Improve/Module/store.html.twig', array(
+            'pageContent' => file_get_contents($this->getAddonsUrl($request)),
+            'layoutHeaderToolbarBtn' => array(),
+            'layoutTitle' => $this->get('translator')->trans('Modules catalog', array(), 'Admin.Navigation.Menu'),
+            'requireAddonsSearch' => true,
+            'requireBulkActions' => false,
+            'showContentHeader' => true,
+            'enableSidebar' => true,
+            'help_link' => $this->generateSidebarLink('AdminAddonsCatalog'),
+            'requireFilterStatus' => false,
+            'level' => $this->authorizationLevel($this::CONTROLLER_NAME),
+        ));
+    }
+
+    private function getAddonsUrl(Request $request)
+    {
+        $psVersion = _PS_VERSION_;
```

Comment:
I don't want to to it, do you agree to declare a new PS_VERSION constant inside AppKernel and register it as container parameter? This constant will have to be updated at every release!